index.html

```diff
@@ -2423,26 +2423,25 @@ <h2>Contexts and Vocabularies</h2>
         <h3>Context Injection</h3>
 
         <p>
-The `@context` property is used by implementations that employ JSON-LD to ensure that implementations are using the
+The `@context` property is used to ensure that implementations are using the
```

Comment:
```suggestion
The `@context` property is used by implementations that employ JSON-LD to ensure that implementations are using the
```
Please do not delete this necessary qualifier.

Reply:
I agree with @selfissued given the text below which says "Implementations that do not intend to use JSON-LD"

Reply:
This has been done in #92; resolving.